acsoo/templates/project/+project.name+/README.rst.bob

```diff
@@ -11,7 +17,7 @@ Initialize virtualenv
 ---------------------
 
 create and activate virtualenv, possibly with virtualenvwrapper's
-``mkvirtualenv odoo-{{{ project.name }}} -a .``
+``mkvirtualenv odoo-{{{ project.name }}} -a . {{{ mk_venv_extra }}}``
```

Comment:
Here it's documentation only, so you can write `$(which {{{ python_version }}} )`

Same in .gitlab.-ci.yml.